src/utils/utils.ts

```diff
@@ -1,14 +1,15 @@
 import { AmountEuroCents } from "../generated/aca/AmountEuroCents";
 import { EntityTypeEnum, NewDebtPositionRequest } from "../generated/aca/NewDebtPositionRequest";
 
-export const createRequestBodyForAca = (debitPositionAmount: AmountEuroCents): NewDebtPositionRequest => ({
-    amount: debitPositionAmount,
+export const createRequestBodyForAca = (
+    debitPositionAmount: Number
+): NewDebtPositionRequest => ({
+    amount: debitPositionAmount as AmountEuroCents,
     description: "Description test",
     entityFiscalCode: "RYGFHD54YR73DF73",
     entityFullName: "Mario Rossi",
     entityType: EntityTypeEnum.F,
     expirationDate: new Date("2030-01-01T09:00:00.000Z"),
     iuv: Math.floor(Math.random() * 1000000000000000).toString(),
```

Comment:
Beware that this method cannot generate all possible IUVs due to floating point accuracy.